src/components/Skeleton/Skeleton.stories.tsx

```diff
@@ -31,29 +31,60 @@ export const Default: StoryObj<Args> = {
   },
 };
 
-export const Circle: StoryObj<Args> = {
+export const Circle: StoryObj<React.ComponentProps<typeof Skeleton.Circle>> = {
+  args: {
+    width: 100,
+  },
   render: (args) => {
-    return <Skeleton.Circle width={100} />;
+    return <Skeleton.Circle {...args} />;
   },
 };
 
-export const Text: StoryObj<Args> = {
+export const Text: StoryObj<React.ComponentProps<typeof Skeleton.Text>> = {
+  args: {
+    width: '30ch',
+    height: '1.5rem',
+  },
   render: (args) => {
-    return <Skeleton.Text height="1.5rem" width="30ch" />;
+    return <Skeleton.Text {...args} />;
   },
 };
 
 export const LayoutExample: StoryObj<Args> = {
-  render: (args) => {
+  render: ({ isAnimating }) => {
```

Comment:
We can use `argTypes` to disable controls for certain props if we want here, i.e.:
```
  argTypes: {
    className: {
      control: false,
    },
    width: {
      control: false,
    },
    height: {
      control: false,
    },
  },
```

Reply:
good call. I thought this was controlled by the type argument to `StoryObj`, but that didn't seem to work